apis/core/lib/ResourceStore.ts

```diff
@@ -147,7 +147,7 @@ export class SparqlStoreFacade implements TripleStoreFacade {
         BIND ( ${now} as ?now )
         BIND ( ${pointer.term} as ?object )
```

Comment:
Is the BIND still needed?

Reply:
In the constructed triples.

I thought it was neat not to repeat the `<URL>` everywhere. Maybe I misunderstand BIND anyway